.github/workflows/Regression.yaml

```diff
@@ -33,3 +33,9 @@ jobs:
           export MODULEPATH=/scratch-local/software/modulefiles
           module load opensn/gcc/12.3.0
           test/run_tests -d test/ -j 32 -v 1 -w 3
+      - name: test tutorials
+        shell: bash
+        run: |
+          export MODULEPATH=/scratch-local/software/modulefiles
+          module load opensn/gcc/12.3.0
+          test/run_tests -d doc/tutorials -j 32 -v 1 -w 3
```

Comment:
This line needs to be: `test/run_tests -d tutorials -j 32 -v 1 -w 3` since the tutorials were moved out of `doc` in the last commit.